wiki/Beatmap/Pattern/Mania/Jacks/en.md

```diff
@@ -0,0 +1,31 @@
+---
+stub: true
+tags:
+  - jacks
+---
+
+# Jacks
+
+Within specific pattern context, **jacks** are typically used to refer to 3 or more consecutive notes within the same column. They are referred to as **jacks** since the motion required to play them mimics the motion of a jackhammer.
```

Comment:
To diferenciate this with an anchor I would add some annotation that jacks tipically are snapped faster than anchors and tend to be shorter.

I would also specify what jacks tipically represent as a pattern (using pitch relevancy or highlighting a quick burst for an instrument or something along those lines)

Reply:
What do you think about this ?
```suggestion
Within pattern specific context, **jacks** are typically used to refer to 3 or more consecutive notes within the same column. They are referred to as **jacks** since the motion required to play them mimics the motion of a jackhammer. Jacks are typically snapped at 1/4th or higher snap intervals and tend to be less notes overall, differentiating it from an [anchor](wiki/pattern/mania/anchors). You will often see jacks represent consecutive repeated sounds that occur within the music.
```
Didn't want to stray too far into specific mapping related techniques but I think this should cover enough of ground